<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="spisok">
          <property name="styleSheet">
           <string notr="true">  color:#2E2F30 ;
  background-color: rgba(0, 5, 0, 30);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinWriting">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Создать запись</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nameOfModel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="category">
            <item>
             <property name="text">
              <string>Профессиональный</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Любительский</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Полупрофессиональный</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="analogOrNot">
            <property name="text">
             <string>Цифровой</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="producer">
            <item>
             <property name="text">
              <string>Nikon</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Panasonic</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sony</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Canon</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Olympus</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Зенит</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="matrixResolution">
            <property name="minimum">
             <double>2.000000000000000</double>
            </property>
            <property name="maximum">
             <double>20.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="changeLens">
            <property name="text">
             <string>Возможность смены объектива</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="size">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="weight">
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>8000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="cost">
            <property name="minimum">
             <number>1000</number>
            </property>
            <property name="maximum">
             <number>150000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="date"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="saveBtn">
              <property name="text">
               <string>Сохранить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="denied">
              <property name="text">
               <string>Отмена</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>939</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
